src/cmd/manage_nsfs.js

```diff
@@ -733,6 +745,42 @@ function is_undefined(value) {
     return false;
 }
 
+async function whitelist_ips_management(args, config_root) {
+    const ips = args.ips;
+    const is_valid = await validate_whitelist_arg(ips);
+    if (!is_valid) {
+        print_whitelist_usage();
+        return;
+    }
+    const whitelist_ips = JSON.parse(ips);
+    const config_path = path.join(config_root, 'config.json');
```

Comment:
please update nsfs_config_schema, we are validating the schema of config.json on config.js, so it should fail without it.

Reply:
@romayalon validation is not failing because of a new property, but still I think its a good to have a validation.